plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Assets.js

```diff
@@ -162,6 +168,49 @@ Ext.define('NX.coreui.controller.Assets', {
         assetContainer = me.getAssetContainer();
 
     assetContainer.refreshInfo(gridView.up('grid').componentModel, assetModel);
+  },
+
+  /**
+   * @protected
+   * Enable 'Delete' when user has 'delete' permission.
+   */
+  bindDeleteAssetButton: function(button) {
+    var me = this,
+        component = me.getAssetList().componentModel;
+    button.mon(
+        NX.Conditions.or(
+            NX.Conditions.isPermitted('nexus:repository-admin:*:*:delete'),
+            NX.Conditions.isPermitted('nexus:repository-admin:' + component.get('format') + ':' +
+                component.get('repositoryName') + ':delete')
```

Comment:
Unsure why we need both of these?  Is there missing logic in the UI ported wildcard handling?

```
nexus:repository-admin:maven2:myrepo:delete
```

is implied by 'nexus:repository-admin:*:*:delete'

Also not sure repository-admin is the right perm, that was intended to delete the repository, not specifically contents inside of it.  

So this is closer to a repository-view delete permission which would map similarly to if you had done an HTTP DELETE?